ci/values/kubetest/test_kafka_service_nodeport.yaml

```diff
@@ -0,0 +1,34 @@
+cloud: local
+
+kafka:
+  enabled: true
+  service:
+    type: NodePort
```

Comment:
Note: I don't think this test gives us anything that a unit test wouldn't give us.

Suggestion: Drop this particular test (but make sure it's covered by unit tests), test connectivity from within pods within the external kafka test.

Reply:
Ditto

Reply:
Removed